src/Kafka/ManageKafkaPermissions/components/ResourcePrefix.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormGroup, TextInput, ValidatedOptions } from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+
+export type ResourcePrefixProps = {
+  value: string | undefined;
+  onChangeValue: (value: string | undefined) => void;
+  invalid: boolean;
+};
+
+export const ResourcePrefix: React.VFC<ResourcePrefixProps> = ({
+  value,
+  onChangeValue,
+  invalid,
+}) => {
+  const { t } = useTranslation(["manage-kafka-permissions"]);
+
+  const onPrefixValueChange = (value: string | undefined) => {
+    onChangeValue(value);
+  };
+
+  return (
+    <FormGroup
+      validated={invalid ? ValidatedOptions.error : ValidatedOptions.default}
+      helperTextInvalid={t("common:required")}
+      fieldId={"resource-prefix-input"}
+    >
+      <TextInput
```

Comment:
The current implementation is using the [typeahead select component](https://www.patternfly.org/v4/components/select#typeahead). Is there a reason for using the `<TextInput>` here instead?

<img width="265" alt="image" src="https://user-images.githubusercontent.com/21063328/158889203-f70e41d0-9798-41d6-979b-4dfe3ed3b112.png">

Reply:
AFAIK, we are not actually suggesting any value there in the existing implementation. Considering that the designs also show a plain text input instead of a select, I asked @suyash-naithani to use a plain text input.

Did I miss that requirement?